### DISCUSSION TITLE
Funktionen im Binding möglich?

### DISCUSSION THREAD

**Discussion by mcuiobroker** [Zefau/ioBroker.jarvis#1976] [Q&A]
@Zefau 
Um Inhalte vom String abzufragen nutzt man z.B.
if (string.indexOf('Fehler')>=0){

}
Ist dies auch in den Bindings vorhanden bzw. möglich?
`{"{0_userdata.0.jarvis.system.saveJarvisDPs.saveAction}.indexOf('Fehler')>=0":{"color":"red"}}`

**Comment by Zefau** [comment]
Ja, Funktionen sind grundsätzlich support. Das Binding wird als JavaScript Funktion geparsed und damit geht grundsätzlich alles.

Ggf. musst du den Wert klammern:
` {"({0_userdata.0.jarvis.system.saveJarvisDPs.saveAction}).indexOf('Fehler')>=0":{"color":"red"}}`
